template/contracts/ERC20Token.sol

```diff
@@ -0,0 +1,10 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract ERC20Token is ERC20 {
+    constructor() ERC20("TOKEN_NAME", "TOKEN_SYMBOL") {
+        _mint(msg.sender, 10000000e18);
```

Comment:
So I'm very new to Solidity, so I apologize in advance if I spam TF out of this PR with questions.

Where is this `_mint()` function declared? I'm assuming it's from the inheritance from the open zeppelin congract ->  `is ERC20` ?

Reply:
The `_mint` function comes from the `ERC20.sol` file we are importing and extending this contract from

Reply:
Great, that's what I had thought :) 🙏

Reply:
@rheinardkorf what's the difference? Where is `_safeMint` defined? I couldn't find it for ERC20 https://github.com/OpenZeppelin/openzeppelin-contracts/search?q=safemint

Reply:
@revmischa I can see that `_safeMint` is for ERC721. Apologies.

For reference: https://docs.openzeppelin.com/contracts/4.x/api/token/erc721#ERC721-_safeMint-address-uint256-